docs/style-guide-for-guide-articles.md

```diff
@@ -1,10 +1,10 @@
 # Style guide for creating and editing Guide Articles
 
-We recommend the following guidelines to writing Guide articles to help you get started contributing and create helpful articles.
+We recommend the following guidelines to get you started contributing and creating helpful Guide Articles.
```

Comment:
I suggest leaving not capitalizing `Articles` and use `Guide articles` throughout.